src/dist/replication/meta_server/duplication/duplication_info.h

```diff
@@ -88,10 +88,7 @@ class duplication_info
     void persist_status();
 
     // if this duplication is in valid status.
-    bool is_valid() const
-    {
-        return _status == duplication_status::DS_START || _status == duplication_status::DS_PAUSE;
-    }
+    bool is_valid() const { return is_duplication_status_valid(_status); }
```

Comment:
The function of 'is_duplication_status_valid'  is just as same as original 'is_valid', why would you write a new function instead of using original 'is_valid'?

Reply:
for a shorter name, and no argument needed.

Reply:
ok